exercises/binary/package.yaml

```diff
@@ -1,12 +1,13 @@
 name: binary
-version: 0.9.0.2 # 2016-07-26
+version: 0.9.0.3
```

Comment:
careful. probably leave the comment as-is?

(as for why it's 0.9, because nobody ever bothered to update it to 1.x because it was deprecated. the script that shows outdated versions just skips it)

Reply:
This comment was added in a85c3dcb883ddccf6463576d061691c529a4075c in 2017-05-06 and suggests that

> Test cases adapted from `exercism/x-common/binary.json` on 2016-07-26

We keep this kind of date information as git metadata.

Since the exercise is deprecated, we no longer update the canonical test data.

Other deprecated exercises don't have comments in the `version` line.

Keeping this comment makes automated version retrieval and bumping harder.

Reply:
Back when this track was following https://github.com/exercism/haskell/issues/522 , that comment would have been required.

The good news is that in choosing not to copy that particular clause into the README, I implicitly decided that 0.9.0.x version numbers are now undefined behaviour, and anything goes. We may do whatever we like with this.

So I leave you this to consider:

The commit message has claimed that a version bump has been performed. I do not perceive "remove a comment" to be within the bounds of the operation "bump version number". I recommend this be remedied by either adding in the commit message a note that the comment was removed, or putting the comment removal in its own commit.

This is actually not so important that I would actually be displeased in any way if things were left as-is, but it's otherwise consistent with how I'd act for changes of greater import

Reply:
Thanks for linking to #522.

I've removed `# 2016-07-26` in a separate commit.

I agree, it's not so important. But I'd like if the versions are easily extracted.